docs/uci_defaults.txt

```diff
@@ -0,0 +1,42 @@
+---
+---
+UCI defaults
+============
+
+== Integrating custom settings through UCI defaults
+
+LEDE allows you to add custom settings by adding scripts into the 'files/etc/uci-defaults/' directory, which
+will then be run after flashing. When keeping settings, this should mean the scripts are run *after* the upgrade
+process flashed the image and appended the saved settings to the JFFS partition (which will be mounted as 
+'/overlay'). The path is identical for the buildroot and the image generator. Scripts should not be executable.
+To ensure your scripts are not interfering with any other scripts, make sure they get executed last by giving them
+a high prefix (e.g. 'zzzz_customisations'). A basic script could look like this:
```

Comment:
I mentioned the zzzz vs 999 just to draw attention to it.  An earlier thread on this suggested that perhaps we should just go and fix packages that don't put a prefix in their files to do so, and then perhaps just not run uci_defaults scripts that don't have a numeric prefix.  See http://comments.gmane.org/gmane.comp.embedded.openwrt.devel/38295 comments from jow at least.